### PR TITLE
Remove cached responses older than 4 mins

### PR DESCRIPTION
**Pull request by peterdudfield**
# Pull Request

## Description

remove old cache (which is copied over from sites api)
had to get `pathy==0.10.3` to get it working

## How Has This Been Tested?

CI tests

- [x] Yes

## Checklist:

- [x] My code follows [OCF's coding style guidelines](https://github.com/openclimatefix/.github/blob/main/coding_style.md)
- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: src/cache.py [hunk 1]
```diff
@@ -52,6 +79,8 @@ def wrapper(*args, **kwargs):  # noqa
             if var in route_variables:
```
Comment (braddf) [inline src/cache.py:79]: Just checking as you mentioned this briefly @peterdudfield, is there anything else you can see in this dict that could be removed to make the keys more reusable/less long? Other than `["session", "user", "request"]`

**Review by braddf — APPROVED** [review]
One comment but otherwise makes sense and LGTM 👍

File: src/cache.py (hunk 1 above)
Comment (peterdudfield) [reply src/cache.py:79]: Thanks, there might be, I'll put them in a different issue as we might be able to get rid of some

Comment (peterdudfield) [reply src/cache.py:79]: https://github.com/openclimatefix/uk-pv-national-gsp-api/issues/319

**Comment by peterdudfield** [comment]
> ## Suspect Issues
> This pull request was deployed and Sentry observed the following issues:
> 
> * ‼️ **RuntimeError: dictionary changed size during iteration** `/v0/solar/GB/gsp/forecast/all/` [View Issue](https://openclimatefix.sentry.io/issues/4905604979/?referrer=github-pr-bot)
> 
> Did you find this useful? React with a 👍 or 👎

whats this @braddf

**Pull request merged**